packages/@divvi/mobile/src/public/types.tsx

```diff
@@ -252,10 +252,11 @@ export interface PublicAppConfig<tabScreenConfigs extends TabScreenConfig[] = Ta
     otaTranslations?: {
       crowdinDistributionHash: string
     }
-    // statsig feature gate defaults
+    // statsig feature gate / dynamic config defaults
     showPositions?: boolean
     showImportTokensFlow?: boolean
     showSwapTokenFilters?: boolean
+    enableSwapFee?: boolean
```

Comment:
To avoid confusion, since there are multiple fees involved.
```suggestion
    enableSwapAppFee?: boolean
```